doc/jsonrpc.md

```diff
@@ -9611,6 +9611,7 @@ Name                    | Optional | Type        | Description
 ----------------------- | -------- | ----------- | -----------
 lvol_name               | Required | string      | UUID or alias of the logical volume to create a snapshot from
 snapshot_name           | Required | string      | Name for the newly created snapshot
+xattr                   | Optional | string map  | Xattr for the newly created snapshot
```

Comment:
nit: xattrs

Reply:
You are right, it should be xattrs. But using the script `rpc.py` you can specify multiply xattrs using the option `--xattr par1=val1 --xattr par2=val2`. So I have to call the option xattr for compatibility with command line script.

Reply:
But looking better, in `rpc.py` I can create an `xattrs` array from multiple `xattr` values.
Let me check, if possible I will rename the variable as you suggests

Reply:
Done